website/yarn.lock

```diff
@@ -0,0 +1,10202 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.

```

Comment:
Does this file need to be checked it?

Reply:
No. Committed by mistake. Have removed now